src/InternalCommand.roc

```diff
@@ -0,0 +1,24 @@
+interface InternalCommand
+    exposes [
+        Command,
+        Output,
+        CommandErr,
+    ]
+    imports []
+
+CommandErr : [
+    ExitStatus I32,
```

Comment:
It looks like this is currently the exit code - is that right? Should we distinguish between exit code and signal in case this process was e.g. SIGKILL'd?

Reply:
Done